http/file_server.ts

```diff
@@ -162,7 +162,7 @@ async function serveDir(
     listEntry.push(
       createDirEntryDisplay(
         info.name,
-        fn,
+        fn.replace(currentDir, ''),
```

Comment:
This is not good. We don't have to replace the dir on each occurence.

Reply:
Thank you for your comment. I'll fix and test it now.

Reply:
Try using `dirPath + "/" + info.name`

Reply:
It would be great to get a test of this failure in `http/file_server_test.ts`...

Reply:
> 
> 
> Try using `dirPath + "/" + info.name`

The original version is `dirPath + "/" + info.name`. However, we get the absulute OS path of the current directory as `dirPath`, which is not correct. We should trim the prefix of `currentDir` here.

Reply:
@zekth We should not modify the variable `dirPath` because there are code using it below ( in line 161, `stat(fn)`), where `stat` needs the full path of a file as its argument.

Reply:
@ry I have added two more test cases. All tests passed.